.rubocop.yml

```diff
@@ -44,6 +44,10 @@ Layout/HashAlignment:
 # Enabled but tweaked cops
 ##########################
 
+# To be tweaked until we find the right balance
+Metrics/MethodLength:
+  Max: 12
+
```

Comment:
If the decision trees will continue to grow, and the Max here grow with it, MethodLength will become useless at keeping other methods in check for us - What do you think to excluding the decisions folder from MethodLength?

Reply:
Yes, the idea is to get to some sensible max, I don't know if it will be 12, or 14, but then anything above will have to be either refactored to make it smaller or when it doesn't make sense as it makes it harder to follow or read the code we add the `disable` annotation like probably in decision trees.

I would say, around 14 maybe 15 lines limit is still pretty manageable. Rubocop by default wants 10 which in my opinion is too low and tends to have the perverse side effect of creating too many small methods that harms readability 👍 
But happy to have something lower!

Reply:
No no, that sounds sensible. 

I thought you meant to increment this number every time the decision tree destination method got longer. All good 👍